wallet-new/test/unit/DeltaCompressionSpecs.hs

```diff
@@ -0,0 +1,141 @@
+{-# LANGUAGE RankNTypes #-}
+module DeltaCompressionSpecs (spec) where
+
+import           Universum
+
+import           Data.Map as M
+import qualified Data.SafeCopy as SC
+import           Data.Serialize (runGet, runPut)
+import           Formatting (bprint)
+import           Formatting.Buildable (build)
+
+import           Test.Hspec
+import           Test.QuickCheck (Arbitrary, arbitrary, resize)
+import           Test.QuickCheck.Monadic (monadicIO, pick)
+
+import           Cardano.Wallet.Kernel.DB.Compression
+import           Cardano.Wallet.Kernel.DB.Spec
+
+import           Test.Pos.Core.Arbitrary ()
+
+{-# ANN module ("HLint: ignore Reduce duplication" :: Text) #-}
+
+data BMap k v = BMap {unMap :: Map k v} deriving (Show, Eq)
```

Comment:
Couldn't we use just a normal map? 🤔

Reply:
I was getting some errors, I think there is no Arbitrary for Maps